source/includes/stages/_30-stages-run-history.md.erb

```diff
@@ -86,7 +89,7 @@ Content-Type: application/json; charset=utf-8
 
 The stage history allows users to list stage instances of specified stage. Supports pagination using offset which tells the API how many instances to skip.
 
-<%= available_since('14.3.0') %>
+<%= available_since('19.3.0') %>
```

Comment:
Should we be changing this?

Reply:
@arvindsv -- I too had this question.
I am not sure with the intention of this version number. 
Does it represent, since when the API is available, or since when the current representation of the API is available?

Would it be possible to show:
```
Available since v14.3.0.
Updated on: v19.3.0`
```

or will it cause more confusion to the end users?

---

**Note**: We are a bit of inconsistent with showing `available_since`. 
Some APIs render since when it is available (Eg: [Pipeline Config](https://api.gocd.org/19.2.0/#get-pipeline-config) )
Some APIs render since when the updated version is available (Eg: [Dashboard](https://api.gocd.org/19.2.0/#get-dashboard) )

Reply:
should we say - 
```
Updated to version x since X.x.x?
Available since v14.3.0.
```

Reply:
Yes, something like that, I think is better.